src/developer/web-api/event-hooks.md

```diff
@@ -0,0 +1,119 @@
+# Event Hooks
+
+## Event Hooks { #webapi_event_hooks }
+
+The event hook API enables users to subscribe to events that occur within DHIS2. Currently, there are two types of events that are published: metadata events for the creation, updating, and deletion of metadata objects, and scheduler events for when any type of internal scheduled job is run, such as analytics.
+
+The endpoint can be found on `/api/eventHooks` and must be enabled in your `dhis.conf` with the
+key `event_hooks.enabled = on` to work.
+
+There are 2 main concept for event hooks, you need to configure the source (which includes the path,
+and the field filters to apply) and one or multiple targets (webhooks, JMS etc).
+
+To illustrate some of these points we will go through a simple webhook that listens on metadata changes for dataElements. And have a webhook target enabled.
+
+```json
+{
+  "id": "bRNvL6NMQXa",
+  "name": "webhook-api-token",
```

Comment:
could you explain the `id` and `name`. 
I presume the `name` can be anything? What about the ID; where does that come from?
Do you send this in a POST request to `/api/eventHooks`? (perhaps without the ID, and that is generated?)